lib/controller.js

```diff
@@ -497,6 +510,7 @@ controller.provisionTessel = function(opts) {
 };
 
 controller.deployScript = function(opts) {
+  opts.authorized = true;
```

Comment:
the static way I did this at the moment is done with the purpose it will be changed by other PRs after deeper thinking about each command. For example - @johnnyman727 is right - some could be an option within the CLI `bin/tessel-2.js`